lang/luaossl/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2021 Siger Yang <sigeryeung@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luaossl
+PKG_VERSION:=20200709
+PKG_RELEASE:=1
+PKG_MAINTAINER:=William Ahern <william@25thandClement.com>, Daurnimator <quae@daurnimator.com>, Siger Yang <sigeryeung@gmail.com>
```

Comment:
You have to be the PKG_MAINTAINER. Since you are then the maintainer of the package in openwrt not the upstream maintainer of luaossl